src/main/webapp/go.js

```diff
@@ -34,8 +38,21 @@ const INVISIBLE_CLASS = 'invisible';
 const puzzleArr = [];
 let destIndex;
 const destArr = [];
+let map;
 
-window.onload = getHunt();
+window.onload = function() {
+  addScriptToHead();
+  getHunt();
+};
+
+/**
+ * Add the Map API key to the head.
+ */
+function addScriptToHead() {
+  const newScript = document.createElement('script');
+  newScript.src = 'https://maps.googleapis.com/maps/api/js?key=' + config.MAP_KEY;
```

Comment:
Please use the `callback` URL param in order to make sure the Maps API has loaded before using it. Perhaps the most straightforward is to set the callback to `createMap` and remove the existing `createMap()` call.

See documentation here: https://developers.google.com/maps/documentation/javascript/overview#Loading_the_Maps_API

@rysimone may be able to help with this if you run into any issues, since he used `callback` in the portfolio project.